subsys/emul/espi/emul_espi_host.c

```diff
@@ -217,6 +217,19 @@ int emul_espi_host_port80_write(const struct device *espi_dev, uint32_t data)
 	return 0;
 }
 
+#ifdef CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION
+int emul_espi_host_set_acpi_shm(const struct device *espi_dev, uint8_t *data,
+				size_t size)
+{
+	struct emul_espi_driver_api *api =
+			(struct emul_espi_driver_api *)espi_dev->api;
+
+	__ASSERT_NO_MSG(api);
+	__ASSERT_NO_MSG(api->set_acpi_shm);
+	return api->set_acpi_shm(espi_dev, data, size);
+}
+#endif
+
 /* Device instantiation */
 static struct emul_espi_device_api ap_emul_api = {
 	.set_vw = emul_host_set_vw,
```

Comment:
This api should be extended to handle the lpc_request and return the mmap pointer.

Reply:
Done.